homework/chapter4/dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM agrigorev/zoomcamp-model:mlops-2024-3.10.13-slim
 
+ENV MLFLOW_TRACKING_URI=http://127.0.0.1:5000
```

Comment:
This is still hardcoded and shouldn't be in your Dockerfile. Create a `.env`(this should be gitignored) and read the tracking URI from there. Here's a StackOverflow link on this 
https://stackoverflow.com/questions/46917831/how-to-load-several-environment-variables-without-cluttering-the-dockerfile/46925953